Ginger/GingerCoreNET/AnalyzerLib/RunSetConfigAnalyzer.cs

```diff
@@ -75,8 +79,69 @@ public static List<AnalyzerItemBase> Analyze(RunSetConfig RSC)
                     }
                 }
             }
+
+            //check all configured mapped data still valid
+            foreach (GingerRunner GR in RSC.GingerRunners)
+            {
+                foreach (BusinessFlow bf in GR.BusinessFlows)
+                {
+                    List<VariableBase> inputVars = bf.GetBFandActivitiesVariabeles(true).ToList();
+                    List<VariableBase> optionalVariables = null;
+                    List<VariableBase> optionalOutputVariables = null;
+                    foreach (VariableBase inputVar in inputVars)
+                    {
+                        bool issueExist = false;
+                        Guid mappedGuid = Guid.Empty;
+                        switch (inputVar.MappedOutputType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)